tests/repository/booking_queries_test.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+import pcapi.core.bookings.factories as bookings_factories
+import pcapi.core.offers.factories as offers_factories
+import pcapi.core.users.factories as users_factories
+from pcapi.repository.booking_queries import get_active_bookings_count
+
+
+class GetActiveBookingsCountTest:
+    @pytest.mark.usefixtures("db_session")
+    def test_return_bookings_quantity_for_venue(self):
+        # Given
+        user = users_factories.UserFactory(email="user@example.com")
+        booking = bookings_factories.BookingFactory(user=user)
```

Comment:
On n'a pas besoin d'un utilisateur spécifique. On pourrait donc directement écrire : 
```suggestion
        booking = bookings_factories.BookingFactory()
```

Idem dans les autres tests plus bas (et aussi sûrement pour les tests dans l'autre module de tests).

(D'autre part, je trouve que ces 3 tests pourraient être fusionnés en un seul test sans trop perdre en lisibilité. Mais c'est peut-être une question de goût.)